src/kbc_modules/offline_fs_kbc/common.rs

```diff
@@ -62,7 +77,12 @@ pub mod tests {
     pub const KID: &str = "foo";
     #[allow(dead_code)]
     pub const KEY: [u8; 32] = *b"passphrasewhichneedstobe32bytes!";
-
+    #[allow(dead_code)]
+    pub const POLICYJSON: [u8; 9] = *b"{\"a\":\"b\"}";
```

Comment:
I would probably just use strings for these and then convert to bytes if needed.